spread.yaml

```diff
@@ -0,0 +1,28 @@
+project: classic-snap
+
+path: /home/spread
+
+backends:
+  qemu:
+    systems:
+      - ubuntu-16.04-64:
+          username: ubuntu
+          password: ubuntu
+      - ubuntu-18.04-64:
+          username: ubuntu
+          password: ubuntu
+
+suites:
+  tests/main/:
+    summary: Integration test for the classic snap
+    prepare: |
+      echo "Prepare image"
```

Comment:
I think we need here a way to restore this env. At least to delete the snaps installed to be able to reuse the machine.